apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonIconTestSection.tsx

```diff
@@ -23,29 +23,19 @@ export const ButtonIconTest: React.FunctionComponent = () => {
 
   return (
     <View style={[stackStyle, commonTestStyles.view]}>
+      {svgIconsEnabled ? <Button icon={{ svgSource: svgProps }} style={commonTestStyles.vmargin} /> : null}
+      {svgIconsEnabled ? <Button primary icon={{ svgSource: svgProps }} style={commonTestStyles.vmargin} /> : null}
       <Button
-        icon={{ fontSource: fontBuiltInProps, width: 100, height: 100, color: '#060' }}
+        icon={{ fontSource: { ...fontBuiltInProps, fontSize: 32 }, color: '#060' }}
         content="Font icon"
         style={commonTestStyles.vmargin}
       />
-      <Button icon={{ fontSource: fontBuiltInProps, width: 100, height: 100 }} content="Font icon" style={commonTestStyles.vmargin} />
-      <Button
-        primary
-        icon={{ fontSource: fontBuiltInProps, width: 100, height: 100 }}
-        content="Font icon"
-        style={commonTestStyles.vmargin}
-      />
-      {svgIconsEnabled ? (
-        <Button
-          primary
-          icon={{ svgSource: svgProps, width: 20, height: 20, color: 'red' }}
-          content="SVG"
-          style={commonTestStyles.vmargin}
-        />
-      ) : null}
+      <Button icon={{ fontSource: fontBuiltInProps }} content="Font icon" style={commonTestStyles.vmargin} />
+      <Button primary icon={{ fontSource: fontBuiltInProps }} content="Font icon" style={commonTestStyles.vmargin} />
       {svgIconsEnabled ? (
-        <Button icon={{ svgSource: svgProps, width: 20, height: 20 }} content="SVG" style={commonTestStyles.vmargin} />
+        <Button primary icon={{ svgSource: svgProps, color: 'red' }} content="SVG" style={commonTestStyles.vmargin} />
```

Comment:
I got rid of the width/height now that we have a default width/height for the icon.